src/calculateRentalCost.js

```diff
@@ -1,10 +1,20 @@
+'use strict';
+
 /**
  * @param {number} days
- *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50; 
```

Comment:
Use a constant for the number of days threshold and the discount amount instead of hardcoding the values. For example, `const LONG_TERM_RENTAL_THRESHOLD = 7;` and `const LONG_TERM_RENTAL_DISCOUNT = 50;`.